.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Thank you for contributing! -->
+
+### Before submitting the PR, please make sure you do the following
+
+- [ ] I have read the [contributing guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
+- [ ] I have checked if there is already a issue, pr or discussion, so I don't create a duplicate
+- [ ] I provided a helpful description in this PR that addresses especially the **what** this PR is solving.
+      _Alternatively I referenced the issue with e.g. `fixes #123`_.
+- [ ] Ideally, I include also test that fails without this PR but passes with it.
+
+### What is the purpose of this pull request? <!-- (put an "X" next to an item) -->
+
+- [ ] Documentation update
+- [ ] Bug fix
+- [ ] Add a CLI option
```

Comment:
What do you think if we simplify the options so the CLI option counts as a New Feature? 
Maybe like this:
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other